tests/test_protocols.py

```diff
@@ -106,3 +113,79 @@ def test_invalid_message(self):
     #     sub = protocol.subscribe('python_test')
     #     for message in sub:
     #         assert message == self.valid
+
+
+class TestValidationErrorPublisher(TestCase):
+    @classmethod
+    def setUp(cls):
+        cls.valid_message = {
+            'schema': 'http://json-schema.org/example/card.json',
+            'familyName': 'foo',
+            'givenName': 'bar',
+        }
+        cls.invalid_message = {
+            'schema': 'http://json-schema.org/example/card.json',
+            'givenName': 'baz',
+        }
+        cls.protocol = Protocol(
+            adapter=MockGoogleAdapter('test-client'),
+            serializer=JSONSerializer(),
+            validator=SchemaValidator())
+
+    def tearDown(self):
+        self.protocol.adapter.clear_messages()
+
+    def test_publish_error_defaults_to_false(self):
+        with self.assertRaises(ValidationError):
+            self.protocol.publish('python_test', self.invalid_message)
+
+    def test_invalid_message(self):
+        topic = 'validation_error'
+        schema = 'http://schema.superbalist.com/events/validation_error/validation_error/1.0.json'
```

Comment:
I know these are public but I don't think we should use our company schema to test.
Maybe we could serve the `validation_error` schema file with gh-pages on this repo?

Reply:
That's a good idea. I'm not a fan either.